internal/types/main.go

```diff
@@ -0,0 +1,19 @@
+package types
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+	"gorm.io/gorm"
+)
+
+type Model struct {
+	ID        uuid.UUID `gorm:"type:uuid;default:uuid_generate_v4()"`
+	CreatedAt time.Time
+}
+
+type ModelAutitable struct {
```

Comment:
```suggestion
type ModelAuditable struct {
```

Is that what you meant?

Reply:
yes, thanks 🤦🏼